src/DataTool.php

```diff
@@ -702,7 +705,11 @@ public function accessLocalField($fieldName)
                 $recSeed = $this->generateSeed($this->module, $fieldName, $this->count);
                 $recData = $this->fields[$fieldName];
                 $recType = (!empty($recData['dbType'])) ? $recData['dbType'] : $recData['type'];
-                return $this->getData($fieldName, $recType, $recData['type'], $recSeed);
+                $val = $this->getData($fieldName, $recType, $recData['type'], $recSeed);
+                if (!empty($val)) {
+                    $this->installData[$fieldName] = $val;
+                }
```

Comment:
It looks like it's not enough to check for empty. If i'm not mistaken we have to duplicate [this](https://github.com/dmulitsa/Tidbit/blob/6eb073d27d156e69a5256a445a97262d4806be1c/src/DataTool.php#L147-L154) logic here. If so, then it would be better to extract [these lines](https://github.com/dmulitsa/Tidbit/blob/6eb073d27d156e69a5256a445a97262d4806be1c/src/DataTool.php#L138-L154) into a separate method `generateFieldData($field, $fieldDef)` and just call it here.